com.unity.multiplayer.mlapi/Runtime/Transports/UNET/UnetTransport.cs

```diff
@@ -41,8 +41,8 @@ public enum SendMode
         private WeakReference temporaryBufferReference;
 
         // Lookup / translation
-        private readonly Dictionary<string, int> channelNameToId = new Dictionary<string, int>();
-        private readonly Dictionary<int, string> channelIdToName = new Dictionary<int, string>();
+        private readonly Dictionary<byte, int> channelNameToId = new Dictionary<byte, int>();
+        private readonly Dictionary<int, byte> channelIdToName = new Dictionary<int, byte>();
```

Comment:
do we actually need these by now?

Reply:
I guess I was too chicken to muck with the logic down and into ConnectionConfig::AddChannel.  @TwoTenPvP I wonder if you could give your thoughts?